### DISCUSSION TITLE
Postgres connection

### DISCUSSION THREAD

**Discussion by oscar-duran** [eauleaf/crudr#4] [Q&A]
I have had try to connect this crud to Postgres database, but I have catched this message: " S6 - Checking if someone else made updates to the tables ..." and the crud has shutdown. 

In other hand:  Another message is: " Unrecognized time zone 'CST6'" in my  local machine.

Finally the catch messages process send: "Warning: Error in dplyr::bind_rows: Can't combine `..1$OBS_ID` <character> and `..2$OBS_ID` <integer>."

Some body can help me about it?

**Comment by eauleaf** [comment]
I think that the time_zone translation might be the problem, so spent the
day working on broadening the time zone translations between R and Postgres
for the 'CST6' and most other timezones.

The other idea I had for the 1st and 3rd errors is that the types in the
'DELTAS' database table and the specified types in the R server aren't
compatible for some reason. The deltas table should all be character
classes except for the timestamp. I've seen the dplyr::bind_rows error when
column types don't match, e.g.

bind_rows(head(iris),mutate(head(iris)))

...no problem

> bind_rows(head(iris),mutate(head(iris), Sepal.Length = as.character(Sepal.Length)))

...Error in `bind_rows()`:

Do you have an example dataset I can look at to reproduce the errors?

**Comment by eauleaf** [comment]
Oscar,

Could you test the newest 'crudr' update with your DB connection? I think
the 'CST6' issue should work now as crudr picks an equivalent timezone when
R doesn't recognize what comes from the DB.
